docs/docs/how-to/styling/using-web-fonts.md

```diff
@@ -2,11 +2,7 @@
 title: Using Web Fonts
 ---
 
-This guide covers how to add web fonts to your Gatsby site.
-
-## Web fonts and Gatsby
-
-Web fonts provide a variety of typography styling options for your site. Hosting your fonts within a Gatsby project increases your site’s speed by up to ~300 milliseconds on desktop and 1+ seconds on 3G connections.
```

Comment:
Would be good to some version of this here to explain the "why"

"Hosting your fonts within a Gatsby project increases your site’s speed by up to ~300 milliseconds on desktop and 1+ seconds on 3G connections."